app/views/candidate_interface/equality_and_diversity/start.html.erb

```diff
@@ -3,21 +3,19 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= form_with model: @choice_form, url: candidate_interface_start_equality_and_diversity_path do |f| %>
-      <%= f.govuk_error_summary %>
-
-      <h1 class="govuk-heading-xl"><%= t('equality_and_diversity.title') %></h1>
-      <p class="govuk-body">The Department for Education (DfE) uses equality and diversity data to improve the teacher training application process and make government policy better.</p>
-      <p class="govuk-body">We use the information you give in this short questionnaire to help prevent discrimination in teacher recruitment. We also share it with your teacher training provider if you accept a place on their course.</p>
-      <p class="govuk-body"><%= govuk_link_to 'Find out how we use and look after your data', candidate_interface_privacy_policy_path %></p>
-      <p class="govuk-body">Completing the questionnaire is optional and will not affect the progress of your application.</p>
-
-      <%= f.govuk_radio_buttons_fieldset :choice, legend: { text: t('equality_and_diversity.choice.title') } do %>
-        <%= f.govuk_radio_button :choice, :yes, label: { text: t('equality_and_diversity.choice.yes.label') }, link_errors: true %>
-        <%= f.govuk_radio_button :choice, :no, label: { text: t('equality_and_diversity.choice.no.label') } %>
-      <% end %>
-
-      <%= f.govuk_submit t('continue') %>
-    <% end %>
+
+    <h1 class="govuk-heading-xl"><%= t('equality_and_diversity.title') %></h1>
+    <p class="govuk-body">The Department for Education (DfE) uses equality and diversity data to improve the teacher training application process and make government policy better.</p>
+
+    <p class="govuk-body">We use the information you give us to help prevent discrimination in teacher recruitment</p>
+
+    <p class="govuk-body">Your answers will not be shown to training providers as they assess your application. We will share your answers if you accept a place on their course.</p>
+
+    <p class="govuk-body"><%= govuk_link_to 'Find out how we use and look after your data', candidate_interface_privacy_policy_path %></p>
```

Comment:
```suggestion
    <p class="govuk-body"><%= govuk_link_to 'Find out how we use and look after your data', candidate_interface_privacy_policy_path %>.</p>
```

Reply:
I wasn't sure about this either. Feels weird to have just a full stop outside the link, so I ended up leaving it as it was. I did try moving "Find out" outside the link and then adding a full stop - which more closely matches the destination page title. But that felt weird too.

Will look at this as part of wider content review of page.